src/cvode/cvode_impl.h

```diff
@@ -33,11 +33,9 @@ extern "C" {
 #endif
 
 #if defined(SUNDIALS_EXTENDED_PRECISION)
-#define RSYM  ".32Lg"
-#define RSYMW "19.32Lg"
+#define RSYM ".32Lg"
```

Comment:
This `RSYM` macro is duplicated a few times, so I'm planning replace it with the single `SUN_REAL_FORMAT`.

Reply:
Take a look at [this branch ](https://github.com/LLNL/sundials/tree/feature/output-format). I'm doing something similar there to clean up `PrintAllStats`. We should coordinate on updating the formatting.